packages/adapter-auto/package.json

```diff
@@ -30,10 +30,11 @@
 		"check": "tsc"
 	},
 	"devDependencies": {
-		"@types/node": "^16.11.68",
-		"typescript": "^4.8.4"
+		"@sveltejs/kit": "workspace:*",
```

Comment:
this dependency is new, is it needed? it's referenced in `index.d.ts` but that was previously taken care of by the `"paths"` in `tsconfig.json`. maybe this way is better, idk, but presumably we'd want to update that config?

Reply:
Here's what I get without it:

![Screenshot from 2022-11-28 12-50-42](https://user-images.githubusercontent.com/322311/204379293-5ca450dc-636d-4b01-8a39-52c5379c2d0f.png)

I just removed the `paths` config instead